nats/src/main/proto/system/ajax/messageproto.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+message ScalarTypes {
```

Comment:
why do you need all of these?

Reply:
i forgot to delete, after completed course about protobuf🥴